{{cookiecutter.project_name}}/tasks.py

```diff
@@ -24,56 +24,62 @@
 
 from invoke import Context, Result, task
 
+NOT_WINDOWS = platform.system() != "Windows"
+
 
 def echo_header(msg: str):
     print(f"\n--- {msg} ---")
 
 
 @dataclass
-class Emo:
-    DO = "🤖"
-    GOOD = "✅"
-    FAIL = "🚨"
-    WARN = "🚧"
-    SYNC = "🚂"
-    PY = "🐍"
-    CLEAN = "🧹"
-    TEST = "🧪"
-    COMMUNICATE = "📣"
-    EXAMINE = "🔍"
+class Msg:
```

Comment:
A bit sad to lose the emojis.

An alternative would be to have it as a class with @property

something like:
```
class Msg:
  def __init__(self):
    self.is_windows = ...
    
  @property
  def PY(self)
    if self.is_windows:
      return ""
    else:
      return "snake emoji"
```

Reply:
Not if it is unicode